### PR TITLE
airlocks no longer spawn an invisible locking forcefield on you

### LINKED ISSUE #13576: Being doorcrushed creates a honest-to-God invisible forcefield around you
https://github.com/d3athrow/vgstation13/blob/465fb8b5b5a00f8697e2e47ce3b02f4ce0bc85bb/code/defines/obj.dm#L339
https://github.com/d3athrow/vgstation13/blob/0dc86433b7dbee2307b6c0113d7e91ff3a90f556/code/game/machinery/doors/airlock.dm#L1291
When getting doorcrushed, a mob spawns a magical "stop" forcefield to hold it in place. For some context, the only other things that use this "stop" forcefield are timestop, and an admin player panel punishment.

For as long as the forcefield is around you, NOTHING MAY MOVE YOU OFF THAT TILE (edit: except with very good timing).

This means nobody may drag you off the door to stop the doorcrush (edit: except with very good timing). The pull will inexplicably break while you remain in place with no indication of why (edit: except with very good timing).
~~This also means that if you are immune to the stun of the doorcrush itself, such as by being a horrorform ling, you will be perfectly standing up yet unable to move. You can turn around, attack anything close enough to you, throw items etc etc. you just may not ever move out of the tile.~~ whoops it turns out that airlocks are FUCKING MAGIC and one of the two things I could find in the game that manage to stun horrorform lings isn't that wonderful

The forcefield lasts for 2 seconds, though obviously it's trivial for anyone doorcrushing you to just keep doorcrushing you which extends the forcefield indefinitely. Hell if someone made an assembly to continuously doorcrush they couldn't even get your corpse off of the tile (edit: except with very good timing).

Why the fuck does this exist?

### PR DESCRIPTION
**Pull request by MadmanMartian**
as requested in the thead. Not entirely sure if I'm using canmove correctly in this case.

closes #13576

### CODE REVIEW

**Comment by ghost** [comment]
What's the point of the canmove if the victim gets stunned anyways?

**Comment by Pathid** [comment]
The explanation for that and the "forcefield" being a thing: https://github.com/vgstation-coders/vgstation13/issues/13576#issuecomment-274309879

**Comment by SonixApache** [comment]
muh doorcrushing

**Comment by Pathid** [comment]
After thinking about this, do we really want to remove doors catching people who are being ZAS'd through them? Or breaking the grip of somebody happening to pull the victim through the door?

The objection of "excellent timing" being necessary to pull someone out falls flat since you can just grab them and spam-click the tile you're on. And as each crush is really only 10 brute you get many tries.

**Comment by ghost** [comment]
Just like use firelock

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1264,15 +1264,11 @@ About the new airlock wires panel:
...
-				var/obj/effect/stop/S = new()
-				S.forceMove(loc)
-				S.victim = L
-
-				spawn (20)
-					qdel(S)
-					S = null
-
 				L.emote("scream",,, 1)
+				L.canmove = 0
```
Comment (PJB3005) [inline code/game/machinery/doors/airlock.dm:1268]: I'm 99% sure that force setting `canmove` won't work for mobs like humans.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

**Comment by Hugh-G-Rection** [comment]
Going full realism mode, the implication here is that someone is being closed inside what is essentially an airtight steel clamp. My thoughts are the damage is way too low, and it opens and closes far too fast. They definitely shouldn't be able to move if an airlock is closed on them, maybe pulled/ripped out at the loss of a limb? If doors open/close time after a crush was slowed to just under stun time, the damage could be increased and AI could still reliably kill people without help, and even with someone there to assist they couldn't get them out of the airlock without opening it (assuming AI would bolt it closed around them) and so would have to delimb the person to free them, likely killing them from bloodloss.

I'm just trying to think of a change that would alter the dynamics of it's use, instead of always being mash as fast as possible. But I'm also just blogposting because I'm bored at work so feel free to ignore me entirely.

**Comment by DamianX** [comment]
Is this going anywhere?

**Pull request closed without merging**